libcipher/utils.py

```diff
@@ -0,0 +1,46 @@
+class EnglishChecker():
+    """This class serves the purpose of checking if a statement is English or not.
+       Two filter methods accomplish this by ensuring:
+        1. At least 85% of the message comprises of english alphabets
+        2. At least 25% of the words in the sentence are contained in
+           the dictionary file, an exhaustive list of english words
+    """
+    def __init__(self):
+        self.UPPERLETTERS = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
+        lower = self.UPPERLETTERS.lower()
+        self.LETTERS_AND_SPACE = self.UPPERLETTERS + lower + ' \t\n'
```

Comment:
This can be obtained from string module.

    string.ascii_letters + string.whitespace